lib/stoplight/traffic_recovery/consecutive_successes.rb

```diff
@@ -49,16 +49,18 @@ def check_compatibility(config)
       #
       # @param config [Stoplight::Light::Config]
       # @param metadata [Stoplight::Metadata]
-      # @return [String]
+      # @return [TrafficRecovery::Decision]
       def determine_color(config, metadata)
         recovery_started_at = metadata.recovery_started_at || metadata.recovery_scheduled_after
```

Comment:
I'm wondering this `|| metadata.recovery_scheduled_after` was added to handle a race condition, when another instance of Stoplight recovered to RED. Example:

- Instances A and B perform recovery simultaneously 
- Instance A recovers to RED (sets recovery_scheduled_after) 
- Instance B tries to receive but does not find `recovery_started_at` value.

Currently, we take `recovery_scheduled_after` instead and perform recovery. Probably it makes sense to PASS in such cases as well. @Lokideos wdyt?